File: azure-intro/csharp/01-iac/code/04-updating-your-infrastructure/step1.cs
using Pulumi;
using Azure = Pulumi.Azure;

class MyStack : Stack
{
    public MyStack()
    {
        var resourceGroup = new Azure.Core.ResourceGroup("my-group");

        var storageAccount = new Azure.Storage.Account("mystorage", new Azure.Storage.AccountArgs
        {
            ResourceGroupName = resourceGroup.Name,
            AccountReplicationType = "LRS",
            AccountTier = "Standard"
        });
    }
}

File: archive/azure-intro/python/01-iac/code/04-updating-your-infrastructure/step1.py
import pulumi
from pulumi_azure import core, storage

# Create an Azure Resource Group
resource_group = core.ResourceGroup('my-group')

storage_account = storage.Account('mystorage',
    resource_group_name=resource_group.name,
    account_tier='Standard',
    account_replication_type='LRS')